src/components/footer/Footer.css

```diff
@@ -0,0 +1,34 @@
+
+.buttons{
```

Comment:
Can this be more descriptive? What if we have multiple kinds of buttons? What if we decide to add some new buttons? Will we change this as well?

Reply:
I don't need this class practiclly, I will delete it. Now I understand reason about naming class.